pkgs/applications/editors/vim/plugins/nvim-treesitter/overrides.nix

```diff
@@ -100,7 +117,7 @@ in
             ln -s ${withAllGrammars}/CONTRIBUTING.md .
             export ALLOWED_INSTALLATION_FAILURES=ipkg,norg,verilog
```

Comment:
```suggestion
```

This was the part I was thinking about. We were skipping certain failures because the nvim version running the test expected them ( they were removed from main)